template-for-repository/negative_tests/float_overflow_check/float_overflow_check_harness.c

```diff
@@ -0,0 +1,12 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+/**
+ * A negative test for --float-overflow-check flag
+ */
+void float_overflow_check_harness() {
+    float overflow, increment;
```

Comment:
Would there be value in also doing this for `double`? Or is it basically the same check?